uportal-war/src/main/data/quickstart_entities/pags-group/Member_of_Students.pags-group.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to Apereo under one or more contributor license
+    agreements. See the NOTICE file distributed with this work
+    for additional information regarding copyright ownership.
+    Apereo licenses this file to you under the Apache License,
+    Version 2.0 (the "License"); you may not use this file
+    except in compliance with the License.  You may obtain a
+    copy of the License at the following location:
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<!--
+  Group member_of_students: "Member of Students (PAGS)"
+  All IPersons that are in the Students Group are members of this group
+-->
+<pags-group script="classpath://org/jasig/portal/io/import-pags-group_v4-1.crn">
+  <name>Member of Students (PAGS)</name>
+  <description>Member of Students group</description>
+  <selection-test>
+    <test-group>
+      <test>
+          <tester-class>org.jasig.portal.groups.pags.testers.AdHocGroupTester</tester-class>
+          <includes>Students</includes>
+          <excludes>BannedStudents</excludes>
+          <excludes>SummerSessionStudents</excludes>
```

Comment:
I don't think we can check in (effectively) foreign-key references to groups that don't exist.  It would be confusing, even if it didn't actually break.